opennebula/resource_opennebula_template_test.go

```diff
@@ -25,6 +25,7 @@ func TestAccTemplate(t *testing.T) {
 					resource.TestCheckResourceAttr("opennebula_template.template", "name", "terra-tpl"),
 					resource.TestCheckResourceAttr("opennebula_template.template", "permissions", "660"),
 					resource.TestCheckResourceAttr("opennebula_template.template", "group", "oneadmin"),
+					resource.TestCheckResourceAttr("opennebula_template.template", "cpu", "0.5"),
```

Comment:
I still need to add some tests, but anyway, for dynamic parts as context or user_inputs it may require a bit more of work are they can take custom values